book_source/adv_user_guide_web/Benchmarking.Rmd

```diff
@@ -0,0 +1,84 @@
+     <variables>
+       <variable_id>411</variable_id> 
+     </variables>
+     <metrics>
+       <metric_id>1000000003</metric_id> 
+     </metrics>
+     <new_run>TRUE</new_run>
+   <reference_run_id>1000000050</reference_run_id>
+ </benchmark>
+```
+
+## 1. New Run
+
+- Setup your run (preferably using the pecan interface). 
+- When the run has completed, navigate to the workflow visualization Shiny app. 
+- Load model data
+  + Make sure that your model output is loading properly (i.e. you can see plots of your data)
+- Load benchmarking data
+  + Again make sure that you can see the uploaded data plotted alongside the model output. In the future there will be more tools for double checking that your uploaded data is appropriate for benchmarking, but for now you need to do the sanity checks by  hand. 
+- Navigate to the Benchmarking tab
+  + The first step is to register the new model run as a reference run in the database. Benchmarking cannot be done before this step is completed. When the reference run record has been created, additional menus for benchmarking will appear. 
+  + Next from the menu select
+    + The variables in the uploaded data that you wish to compare with model output. Note: this list shows all available variables regardless of whether or not there is a comparable variable in the model output. Thus if you select a variable to be benchmarked and it does not show up in the results, double check that it is also in the model output! 
+    + The numerical metrics you would like to use in your comparison. 
+    + Additional comparison plots that you would like to see.
+  + All these selections populate the benchmarking section of the xml witch is then saved in the same location as the original run output. 
+- Visualizations of benchmarking results are in progress. 
+  + Currently all results are stored in the `benchmarking.output.Rdata` object which is stored in the same folder as the original model run. This loads a list  called `results`  which contains the following:
+    + `input` a list element for each input file that was used for benchmarking
+      - `bench.results`: a data frame of all numeric benchmarking scores
+      - `format`: a data frame that can be used to see how the input data was transformed to make it comparable to the model output. This involves converting from the original variable names and units to the internal pecan standard. 
+      - `aligned.dat`: a data frame of the final aligned model and input values. 
+  + All plots are saved as pdf files (with names beginning with "benchmarking.")
+
+## 2. Clone Run
+
+It may be the case that you want to exactly recreate a run that was done on a different server. Or that you want to take the settings that were used to set up a run of ED2 at Harvard Forrest and use them to set up a run at the same site but using SIPNET instead. For both of these cases, instead of recreating the runs by hand every time, we are developing a way to "clone" runs. 
```

Comment:
1) For the first sentence, why would you have to 'clone' a benchmark from a different server just to rerun it? Seems like that should work just by providing the reference run id. Admittedly at the moment it might fail if it doesn't have the required inputs or data, and can't get them automatically, but that's a completely separate issue having to do with moving files among servers.

2) In the second sentence, I'd describe the general concept (you want to repeat a run done in one model with another model) then use HF ED2 -> SIPNET as an example.

That said, this whole section seems to convolve cloning runs and cloning benchmarks. The former probably needs to be documented elsewhere and in greater detail (the text below gives no indication HOW to run 'the cloning workflow') so that this section can focus on cloning benchmarks (but can link to the more general doc on cloning runs).